gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/Orchestrator.java

```diff
@@ -109,10 +109,10 @@ public class Orchestrator implements SpecCatalogListener, Instrumentable {
 
   private final ClassAliasResolver<SpecCompiler> aliasResolver;
 
+  @Inject
   public Orchestrator(Config config, Optional<TopologyCatalog> topologyCatalog, Optional<DagManager> dagManager,
```

Comment:
I'd argue that, similarly to the `EventSubmitter` the `DagManager` is no longer justifiably `Optional`.  while we're at it, can we clean this up too?

also, what of the `Optional<TopologyCatalog>`?